src/Helpers/APIHelper.js

```diff
@@ -235,3 +235,18 @@ export function getSystemsIds(apiProps) {
     let result = api.getSystemsIds(...parameterArray);
     return result;
 }
+
+export function getAnnouncement() {
+    let result = api.getAnnouncement();
+    return result;
+}
+
+export function getDashbar(apiProps) {
+    let parameterNames = [
+        ...defaultParams,
+        ...systemsParams
+    ];
```

Comment:
This `parameterNames` variables should be a list of names of params which are passed with `getDashbar` request. Currently this endpoint does not have any params. In the near future we will have these params:
![Screenshot from 2022-03-31 17-16-14](https://user-images.githubusercontent.com/8426204/161090043-3b1d719e-2351-4f5c-8ffd-c0596a45b552.png)

So you can either do this because we don't have any params
```
export function getDashbar() {
    let result = api.getDashbar();
    return result;
}
```

Or prepare the structure for the future params by:
```
export function getDashbar(apiProps) {
    let parameterNames = [
       'tags',
       'sap_sids',
       'sap_system'
    ];
    let parameterArray = constructParameters(apiProps, parameterNames);
    let result = api.getDashbar(...parameterArray);
    return result;
}
```

Be careful, order in `parameterNames` matters.